### PR TITLE
Add `borrow_all()` to `SingleThreadedComments`

### PR DESCRIPTION
**Pull request by dsherret**
I decided to go ahead and work on this because it's very simple. Let me know about any changes you'd like or if you even want something like this. Thanks!

From #1261

### CODE REVIEW

File: common/src/comments.rs [hunk 1]
```diff
@@ -107,7 +111,8 @@ where
     delegate!();
 }
 
-pub type SingleThreadedCommentsMap = Rc<RefCell<FxHashMap<BytePos, Vec<Comment>>>>;
+pub type SingleThreadedCommentsMapInner = FxHashMap<BytePos, Vec<Comment>>;
+pub type SingleThreadedCommentsMap = Rc<RefCell<SingleThreadedCommentsMapInner>>;
```
Comment (dsherret) [inline common/src/comments.rs:115]: I'm not sure about this naming, but I didn't want to cause a breaking change.

Comment (kdy1) [reply common/src/comments.rs:115]: It seems ok to me.

**Comment by dsherret** [comment]
Thanks for the quick merge! I upgraded and will start adding comment related functionality now.

**Comment by kdy1** [comment]
No problem. Thanks for the pr.

**Pull request merged**